spec/support/mock_api.rb

```diff
@@ -0,0 +1,48 @@
+module AnsibleTowerClient
```

Comment:
I did more digging myself - and although I'd like to see our mocking and testing libraries under a separate namespace, we currently have the `FactoryHelper` under the `AnsibleTowerClient` namespace [here](https://github.com/ansible/ansible_tower_client_ruby/blob/master/spec/support/factory_helper.rb#L3-#L4).  So in the interest of consistency let's keep things consistent.